src/components/Button/Button.tsx

```diff
@@ -2,14 +2,15 @@ import React from 'react'
 import styled from 'styled-components'
 import { fonts } from '../../globalStyles/fonts'
 import PendingIndicator from '../PendingIndicator/PendingIndicator'
-import { mediaQueriesPx } from "../../mediaQueries";
+import { mediaQueriesPx } from '../../mediaQueries'
 
 export interface ButtonProps {
   label?: string
   isPrimary?: boolean
   disabled?: boolean
   isLoading?: boolean
   icon?: JSX.Element
+  type: string
```

Comment:
for better type checking lets use string literal type
`type: 'button' | 'other' | 'possible' | 'types' | 'go' | 'here' `

Reply:
anyway I can not see `type` being used anywhere

Reply:
Semantically, we should apply the button type.